### ISSUE TITLE
Exception raised during analysis

### ISSUE THREAD

**Issue by elmjag** [ConorFWild/pandda_2_gemmi#46]
When I try to run pandda analysis, after the while processing stops with:

```
╭───────────────────── Traceback (most recent call last) ──────────────────────╮
│                                                                              │
│ /home/elmjag/pandda2/pandda_2_gemmi/analyse.py:609 in process_pandda         │
│                                                                              │
│   606 │   │   │   │   │   process_autobuilds = process_global                │
│   607 │   │   │   │                                                          │
│   608 │   │   │   │   time_autobuild_start = time.time()                     │
│ ❱ 609 │   │   │   │   autobuild_results_list: Dict[EventID, AutobuildResult] │
│   610 │   │   │   │   │   [                                                  │
│   611 │   │   │   │   │   │   Partial(                                       │
│   612 │   │   │   │   │   │   │   autobuild_func,                            │
│ /home/elmjag/pandda2/pandda_2_gemmi/pandda_gemmi/pandda_functions.py:40 in   │
│ process_local_serial                                                         │
│                                                                              │
│     37 def process_local_serial(funcs):                                      │
│     38 │   results = []                                                      │
│     39 │   for func in funcs:                                                │
│ ❱   40 │   │   results.append(func())                                        │
│     41 │                                                                     │
│     42 │   return results                                                    │
│     43                                                                       │
│                                                                              │
│ /home/elmjag/pandda2/pandda_2_gemmi/pandda_gemmi/common/partial_func.py:8 in │
│ __call__                                                                     │
│                                                                              │
│   5 │   │   self.kwargs = kwargs                                             │
│   6 │                                                                        │
│   7 │   def __call__(self, *args, **kwargs):                                 │
│ ❱ 8 │   │   return self.func(*args, *self.args, **kwargs, **self.kwargs)     │
╰──────────────────────────────────────────────────────────────────────────────╯
```

See attached files below for full log.
[stdout.log](https://github.com/ConorFWild/pandda_2_gemmi/files/8262213/stdout.log)
[stderr.log](https://github.com/ConorFWild/pandda_2_gemmi/files/8262211/stderr.log)

The analysis is launched with following arguments:

```
python analyse.py \
	--autobuild no \
	--cif_strategy default \
	--data_dirs /data/staff/biomax/elmir/pandda2/ \
	--mtz_regex=final_rfill.mtz \
	--pdb_regex=final.pdb \
	--out_dir out \
	--local_processing serial
```

Let me know if I should provide more details to diagnose this issue.

**Comment by ConorFWild** [comment]
Should be fixed as of now in #47 ! Lmk if it works on your side and I'll close if so @elmjag ! ^^

**Comment by elmjag** [comment]
The fix did the trick, thanks! Feel free to close the issue @ConorFWild
